vllm/v1/core/kv_cache_manager.py

```diff
@@ -129,6 +129,11 @@ def __init__(
         self.num_kv_cache_groups = len(kv_cache_config.kv_cache_groups)
         self.block_pool = self.coordinator.block_pool
         self.kv_cache_config = kv_cache_config
+        # Pre-constructed KVCacheBlocks with no blocks, callers should use this
+        # via create_kv_cache_blocks instead of creating new ones to avoid GC
+        # overhead.
+        self.empty_block_list = KVCacheBlocks(
```

Comment:
```suggestion
        self.empty_kv_cache_blocks = KVCacheBlocks(
```

Prefer this name a little bit (the older one is also fine)

Reply:
Updated